src/components/ResourceList/components/Item/Item.tsx

```diff
@@ -321,6 +321,7 @@ export class Item extends React.PureComponent<CombinedProps, State> {
 
   @autobind
   private handleClick(event: React.MouseEvent<any>) {
+    stopPropagation(event);
```

Comment:
I am not sure if this going to break anyones code 😬

Reply:
Hmmm I don't _think_ anyone is wrapping `Resource.Item`s, that would likely break the UI of the list. When this is ready, I can tophat in `web` and grep for all instances of ResourceList and double check that things are working as expected in the lists.